snuba/environment.py

```diff
@@ -26,3 +27,11 @@ def setup_sentry() -> None:
         integrations=[FlaskIntegration(), GnuBacktraceIntegration()],
         release=os.getenv("SNUBA_RELEASE"),
     )
+
+
+clickhouse_rw = ClickhousePool(settings.CLICKHOUSE_HOST, settings.CLICKHOUSE_PORT)
+clickhouse_ro = ClickhousePool(
+    settings.CLICKHOUSE_HOST,
+    settings.CLICKHOUSE_PORT,
+    client_settings={"readonly": True},
+)
```

Comment:
Do these open the connections to clickhouse when we create them or when we use them? If it happens when we create them we will always be using twice as many connections than we need. 
Said that we were already creating both of them when we loaded the query file, still maybe useful to investigate, we could remove useless connections from clickhouse.

Reply:
They are created dynamically as needed: https://github.com/getsentry/snuba/blob/eb772cdb68061619bb0c60a066edfcaf666b4fb7/snuba/clickhouse/native.py#L34-L38